### ISSUE TITLE
trap errors more gracefully?

### ISSUE THREAD

**Issue by amyrebecca** [zooniverse/aggregation-for-caesar#31]
It may or may not be worth changing, but currently submitting a malformed request gets you a 500 page with a ton of detail about the error, including stack traces and some kind of WSGI console login. This could be useful for debugging but I wonder if we should clean it up somewhat for production?

**Comment by CKrawczyk** [comment]
I agree.  Also the current page shown is only because `debug` mode is turned on (https://github.com/zooniverse/aggregation-for-caesar/blob/master/routs.py#L58). 

Side note, this can be done in flask by adding something like
```python
@application.errorhandler(500)
def internal_error(error):
    return "500 error"
```
to the application and turning off debug mode.
